### PR TITLE
have the high score expire after 24hrs, tweaks to stats display

### PR DESCRIPTION
**Pull request by railsagainstignorance**

### CODE REVIEW

File: bin/lib/game.js [hunk 1]
```diff
@@ -381,18 +386,45 @@ class Game{
...
+				}
 			}
+
 			GAMES_STATS.scoreCounts[score] += 1;
-			GAMES_STATS.maxScore = Math.max(GAMES_STATS.maxScore, score);
+
+			if (score > GAMES_STATS.maxScore) {           // new high score!
+				GAMES_STATS.maxScoreSetMillis = nowMillis;
+				GAMES_STATS.maxScore = score;
+			} else if( score === 0 ){                     // yeah well, best forgotten
```
Comment (seanmtracey) [inline bin/lib/game.js]: Do we need this line?

File: routes/index.js [hunk 2]
```diff
@@ -174,7 +174,22 @@ router.get('/answer', S3O, (req, res) => {
...
+		debug(JSON.stringify(err));
+		res.json({
+			status : 'err',
+			message : 'An error occurred as we tried to get the stats.'
+		});
+	})
+	;
+});
+
+router.post('/stats', S3O, (req, res) => {
```
Comment (seanmtracey) [inline routes/index.js]: Is this just for S3O?

File: bin/lib/game.js (hunk 1 above)
Comment (railsagainstignorance) [inline bin/lib/game.js]: removes the need for subsequent logic along the lines of (score > 0 && )

File: routes/index.js (hunk 2 above)
Comment (railsagainstignorance) [inline routes/index.js]: yes

**Pull request merged**